packages/grafana-e2e/src/flows/addDataSource.ts

```diff
@@ -4,7 +4,10 @@ export const addDataSource = (pluginName?: string): string => {
   pluginName = pluginName || 'TestData DB';
   e2e().logToConsole('Adding data source with pluginName:', pluginName);
   e2e.pages.AddDataSource.visit();
-  e2e.pages.AddDataSource.dataSourcePlugins(pluginName).click();
+  e2e.pages.AddDataSource.dataSourcePlugins(pluginName)
+    .scrollIntoView()
```

Comment:
Find this to be kind of interesting, the `Test Data` data source is under the fold so to be certain that it's visible we need to call `scrollIntoView`. But this isn't true for all test runs, as the majority of the test runs have passed without this.